www/src/public/js/controllers/builder/main.js

```diff
@@ -178,11 +212,14 @@
                             case 2:
                             case 3:
                                 var newList = createListFromStringV2(listStrs[i], listVersion);
-                                break;                            
+                                break;    
+                            case 4: //should we change how this works to use the new built in version?
+                                var newList = createListFromStringV2(listStrs[i], listVersion);
+                                break;
                             default:
                                 break;
                         }
-
+						
```

Comment:
idk why you get all these weird spacing issues... verrrry bizarre

Reply:
Maybe because I use tab instead of spaces? I just noticed this:

![image](https://user-images.githubusercontent.com/100298859/219495867-5b0a3a2d-cece-44f0-b81f-ea170283865e.png)

Reply:
yeaa, that's not ideal, there should be a setting that auto converts tabs to spaces. It should end up making 1 tab = 4 spaces. That way you can continue to use tabs, but it won't impact the file in a weird way